src/rt/cast_.d

```diff
@@ -53,7 +53,12 @@ void* _d_interface_cast(void* p, ClassInfo c)
     if (!p)
         return null;
 
-    Interface* pi = **cast(Interface***) p;
+    Interface* pi;
+    if (auto pi_ = *cast(Interface***)p)
+    {
+        pi = *pi_;
+    }
+    if (!pi) return null;
```

Comment:
Sorry that was a copy paste. I use a mix of styles depending on the length of the line. Reasoning is: if it reads well in one line, why not? If not, might as well use the `{ }` as well to make it obvious it's not a one liner.